### ISSUE TITLE
MISO_SASHIMI error: Could not find MISO output files

### ISSUE THREAD

**Issue by amizeranschi** [nf-core/rnasplice#89]
### Description of the bug

Opening a new report as requested by @jma1991

The issue is described here and in subsequent messages: https://github.com/nf-core/rnasplice/issues/85#issuecomment-1665059633

The command and input files are listed in the first message here: https://github.com/nf-core/rnasplice/issues/85

**Comment by amizeranschi** [comment]
Hello @jma1991, is there any progress regarding this issue?

**Comment by amizeranschi** [comment]
I have now tested the recent changes via `zifornd/rnasplice -r miso_fix` and can confirm that they fix this issue.

@jma1991 how close are you to a `v1.0.0` release of the pipeline? I'm planning to use it in a study and it would be nice to have a proper release version to reference in the paper.

**Comment by jma1991** [comment]
Hey @amizeranschi - Glad to see that fixed your issue.

Regarding a v1 release, we're just waiting on one last reviewer to complete their second review. I'm not sure when they will be available to complete their review, but hopefully will be soon.

**Comment by jma1991** [comment]
@amizeranschi We have now officially released v1 of the pipeline. There is a Zenodo citation you can use in your paper now.
